content/00.front-matter.md

```diff
@@ -4,9 +4,21 @@
 
 {% for author in authors %}
 + **{{author.full_name}}**<br>
-  [\@{{author.github_username}}](https://github.com/{{author.github_username}}) ·
-  [{{author.orcid}}](https://orcid.org/{{author.orcid}}) ·
-  {{author.affiliations}} ·
-  Funded by {{author.funders}}
+  {% set previous = false %}
```

Comment:
@dhimmel That sounds reasonable. I can make the ORCID as first but not enforce it. This will simplify the code.